Detectors/Base/src/MaterialManager.cxx

```diff
@@ -12,8 +12,8 @@
 /// \brief Implementation of the MaterialManager class
 
 #include "DetectorsBase/MaterialManager.h"
-#include <TVirtualMC.h>  // for TVirtualMC, gMC
-#include "TString.h"     // for TString
+#include "TVirtualMC.h"
```

Comment:
why these changes? They are not necessary for this PR, I guess.

Reply:
hum, no, indeed. In the original commit that include was no longer needed, then I put it back in the second commit but as a quotes include instead of angled include, out of habit I guess (as I've never quite understood why is Root considered a system include ;-) )